docs/reference/aggregations/bucket/composite-aggregation.asciidoc

```diff
@@ -348,6 +348,34 @@ GET /_search
 \... will sort the composite bucket in descending order when comparing values from the `date_histogram` source
 and in ascending order when comparing values from the `terms` source.
 
+====== Missing bucket
```

Comment:
This seems to have 1-2 extra `=` signs, and isn't rendered properly/well:
https://www.elastic.co/guide/en/elasticsearch/reference/6.4/search-aggregations-bucket-composite-aggregation.html